includes/collection/class-remote-actors.php

```diff
@@ -536,6 +536,11 @@ public static function normalize_identifier( $actor ) {
 	 */
 	public static function get_public_key( $key_id ) {
 		$actor = get_remote_metadata_by_actor( \strip_fragment_from_url( $key_id ) );
```

Comment:
We should also replace this with the local lookup, because now we try to resolve it from the remote resource, try to store it, and then fail, to then try to resolve it again.